src/com/content-event/event-slaughter.js

```diff
@@ -178,56 +180,85 @@ export default class ContentEventSlaughter extends Component {
 
 		return (
 			<div class="-item">
-				<div class='-text' title={idea}>{idea}</div>
+				<div class="-text" title={idea}>{idea}</div>
 			</div>
 		);
 	}
 	renderMyIdeas() {
 		return Object.keys(this.state.ideas).map(this._renderMyIdea);
 	}
 
-	renderBody( {current, votes, ideas, done/*, error*/} ) {
+	renderBody( state ) {
+
+		let seen = Object.keys(state.votes).length;
+
+		var labels = [
+			'slaughtered',
+			'kept',
+			'left'
+		];
+
+		var values = [
+			0,
+			0,
+			state['votes-left']
+		];
+
+		if ( seen != 0 ) {
+			Object.values(state.votes).forEach((v) => {
+				if ( v == 0 ) {
+					values[0]++;
+				}
+				else if ( v == 1 ) {
+					values[1]++;
+				}
+				// else if v=2 it's flagged
+			});
+		}
+
 		var StatsAndDetails = (
-			<div>
+			<div class="history">
 				<h3>Recent Themes</h3>
 				{this.renderRecentQueue()}
 			</div>
 		);
 
-		if ( done ) {
+		if ( state.done ) {
 			return (
-				<div>
-					<div>Wow! {"You're totally done!"} Amazing! You slaughtered {Object.keys(votes).length} themes!</div>
-					{StatsAndDetails}
+				<div class="-stats">
+					<div>Wow! {"You're totally done!"} Amazing! You slaughtered {Object.keys(state.votes).length} themes!</div>
```

Comment:
This is probably not yours, but why is that a string in curlies?

Reply:
I think it might be so that the punctuation doesn't need to be escaped, but that is only a guess?

Reply:
oh I think it is "worse" than that, come to think about it. Probably only to make editor not falsly misinterpret it though it is perfectly valid

Reply:
Yeah I'm using an editor that really doesn't understand JSX. Now that I'm sharing though, we should disregard stuff that's broken in my editor and go with what looks right.

Reply:
Removed enclosing `{""}`